integration-tests/fromMnemonic-send-tez.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { OperationContentsAndResultTransaction } from "@taquito/rpc";
+import { InMemorySigner } from "@taquito/signer";
+import { TezosToolkit } from "@taquito/taquito";
+import { CONFIGS } from "./config";
+import * as Bip39 from 'bip39';
+
+CONFIGS().forEach(({ lib, rpc, setup }) => {
+  let Funder: TezosToolkit;
+  describe(`Test fromMnemonic instantiation with rpc: ${rpc}`, () => {
+    let mnemonic: string;
+
+    beforeAll(async (done) => {
+      mnemonic = Bip39.generateMnemonic();
+      Funder = lib;
+      await setup();
+      done();
+    });
+    it('Should create an InMemorySigner (ed25519) with the fromMnemonic method and transfer tez to an account', async (done) => {
+      const Tezos = new TezosToolkit(rpc);
+      // with all default values
+      const signer = InMemorySigner.fromMnemonic({ mnemonic });
+      Tezos.setSignerProvider(signer);
+
+      const funderPKH = await Funder.wallet.pkh();
+      const tezosPKH = await Tezos.wallet.pkh();
+
+      const fundOp = await Funder.wallet.transfer({ to: tezosPKH, amount: 5 }).send();
```

Comment:
By changing `beforeEach` to `beforeAll` the test started failing on kathmandunet, because it tries to spend too many tokens from the key retrieved from the keygen (limit is 10). Changing 5 to something else, like 2 in the 3 tests will fix that.